tests/fixtures/issues/issue-096.php

```diff
@@ -3248,7 +3248,7 @@
             'MinorVer' => '0',
             'Platform' => 'JAVA',
             'Platform_Version' => 'unknown',
-            'Platform_Description' => 'Samsung Bada',
+            'Platform_Description' => 'unknown',
```

Comment:
This is the result of adding the platform_description value to the JAVA platform.

This shouldn't be "Samsung Bada" since the platform is "JAVA".  There are some more like this, as you'll see, but the others are all "Android OS" (but should be "unknown", which is what JAVA's platform_description is).